doc/longtermstorage.md

```diff
@@ -169,7 +169,7 @@ Refer to the steps below to add ECS server certificate or CA's certificate into
     $ kubectl create -f ecs-tls.yaml
     ```
 
-3. In Pravega manifest, add the secret name defined above into "tls/static/caBundle" section. 
+3. In Pravega manifest, add the secret name defined above into "tls/static/caBundle" section.
```

Comment:
This line shows as changed but it has exactly the same content as earlier. Please revert the change.

Reply:
done